### ISSUE TITLE
Ability to search by Layer, Coverage and body parts for armor and clothing

### ISSUE THREAD

**Issue by utechenko-v** [nornagon/cdda-guide#107]
For example, I'm viewing info on tuxedo and see that it covers Legs (Thighs, Knees, Lower Legs, Hips). Then I want to see what other clothes can cover Lower Legs, to find the best protection or warmth for Lower Legs.

**Comment by nornagon** [comment]
Yeah I love this idea. I'd like to build a much better system for searching and sorting through clothing and armor, but it's a pretty big project.
